PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -283,6 +286,27 @@
                 }
             }
 
+            // if view in photo library picker controller, always mask the whole view
+            if let pickerViewController = view.nearestViewController as? UIImagePickerController {
+                maskableWidgets.append(pickerViewController.view.toAbsoluteRect(window))
+                return
+            }
```

Comment:
I think this should be masked by default, but we should offer a way out.
I do know a user that wanted to see the picture because the app was all about that.
Maybe a new flag config `maskNativeImagePicker` or something like that?

Reply:
if we can distinguish the album itself from the photo that the user has chosen or taken, I agree, if it's not possible, then it'd say it should be an opt-in feature (masked by default) otherwise we are just blocking this type of usage with no way out.

Reply:
eg: a photo app would need to see what's happening otherwise session replay isn't useful for this type of app, even if it's sensitive, they can always add this under their terms & conditions and make this "possible".